src/components/form/HdRange.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <div
+    :class="fieldClasses"
+    class="range field"
+  >
+    <input type="range"
+      ref="input"
+      v-model="currentValue"
+      :id="name"
+      :name="name"
+      :required="required"
+      :disabled="disabled"
+      :min="minValue"
+      :max="maxValue"
+      :step="rangeStep"
+      @input="updateRangeDecoration"
+      @change="isLegacy ? updateRangeDecoration : null"
```

Comment:
I would add a test for this guy as well. As if `isLegacy` is set it won't call `updateRangeDecoration`.

Reply:
You could also add it to the story so we could play with it as well.

Reply:
Nice! I wrote the test and got that that feature was not working. I made it differently and works great. That's TDD's best flavour. Thank you, man.

I also wrote tests for the other listeners. Big Up!